Segment-Facebook-App-Events.podspec

```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0'
-  s.dependency 'FBSDKCoreKit', '~> 4.15'
+  s.dependency 'Analytics'
+  s.dependency 'FBSDKCoreKit'
```

Comment:
Can we add the dependency targets to the .podspec file for 'Analytics' and 'FBSDKCoreKit' here as well?